src/components/Navbar.js

```diff
@@ -0,0 +1,165 @@
+import { useContext, useState } from "react";
+import { AiOutlineDown, AiOutlineUp } from "react-icons/ai";
+import { BsSearch } from "react-icons/bs"
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+import LoginContext from "../contexts/LoginContext";
+
+export default function Navbar() {
+    const {setUserData, setConfig} = useContext(LoginContext);
+    const [hidden, setHidden] = useState(true);
+    const [search, setSearch] = useState();
+    const navigate = useNavigate();
+
+    function logOut() {
+        localStorage.clear("linkr");
+        setUserData(null);
+        setConfig(null);
+        navigate("/");
+    };
+
+    return (
+        <>
+            <WrapperNavbar>
+                <h1 onClick={() => navigate("/timeline")}>linkr</h1>
+                <WrapperMenuLogOut onClick={() => setHidden(!hidden)}>
+                    {hidden ? (
+                        <AiOutlineDown 
+                            style={{
+                                color:  "#ffffff",
+                                cursor: 'pointer',
+                                
+                            }}
+                            size={30}
+                        />
+                    ) : (
+                        <AiOutlineUp 
+                            style={{
+                                color:  "#ffffff",
+                                cursor: 'pointer',
+                                
+                            }}
+                            size={30}
+                        />
+                    )}
+                    <img src="https://i.pinimg.com/736x/c5/a9/68/c5a968eb0a92b427ca26646cf55526bb.jpg" alt="img"></img>
```

Comment:
**Suggestion:** I would save this image in another file and import into that component

Reply:
Ops, thanks for the heads up!!